runbot_travis2docker/models/runbot_build.py

```diff
@@ -185,14 +185,15 @@ def _get_ssh_keys(self):
             return
         keys = ""
         for own_key in ['author', 'committer']:
+            login = response.get(own_key, {}).get('login')
```

Comment:
If the response[own_key] is None you will have a NoneType error.
You can use
(response.get(own_key) or {}).get(...

Reply:
Yes, you're right, altough it would be `AttributeError`.

There is another possible issue, the key might not be a dictionary, which would also raise `AttributeError`, so I think I will enclose that sentence in the below try-catch block.

Reply:
@moylop260  
I did: `login = response.get(own_key).get('login')`
and added `AttributeError` to the try-except. What do you think?

I'd like to add, I think none of those possibilities should be possible, unless GitLab API is broken. But, since it's a json, `None` and an empty list could be theoretically possible values.